scripts/qa_crowbarsetup.sh

```diff
@@ -1747,6 +1747,10 @@ function custom_configuration()
             ;;
             ceilometer)
                 proposal_set_value ceilometer default "['deployment']['ceilometer']['elements']['ceilometer-cagent']" "['$sles12controller']"
+                proposal_set_value ceilometer default "['deployment']['ceilometer']['elements']['ceilometer-server']" "['$sles12controller']"
+                if [[ -n $deployswift ]] ; then
+                    proposal_set_value ceilometer default "['deployment']['ceilometer']['elements']['ceilometer-swift-proxy-middleware']" "['$sles12controller']"
```

Comment:
Isn't that done automatically? (ie, swift-proxy nodes automatically getting the role on proposal creation)